docs/html/conf.py

```diff
@@ -26,11 +25,13 @@
     "sphinx_copybutton",
     "sphinx_inline_tabs",
     "sphinxcontrib.towncrier",
+    "sphinx_issues",
 ]
 
 # General information about the project.
 project = "pip"
 copyright = "The pip developers"
+issues_default_group_project = "pypa/pip"
```

Comment:
Please move this in its own section with it's own "Options for sphinx_issues" header, so that it's consistent with the rest of the configuration for this proejct.